anesthetic/read/cobaya.py

```diff
@@ -20,7 +20,12 @@ def read_paramnames(root):
     with open(root + ".1.txt") as f:
         header = f.readline()[1:]
         paramnames = header.split()[2:]
-        return paramnames
+        try:
+            s = loadMCSamples(file_root=self.root)
+            tex = {p.name: '$' + p.label + '$' for p in s.paramNames.names}
+            return paramnames, tex
+        except NameError:
+            return paramnames, {}
```

Comment:
This is not actually working, yet. `loadMCSamples` was not imported and `self` does not exist, so this was silently failing. Probably better to put `import getdist` into the try block and only allow failing with `ImportError`.

Reply:
Agreed. With the latest merge of master this is now corrected and tested.